internal/host/validator.go

```diff
@@ -851,29 +850,40 @@ func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRole
 					}
 					if role == host.Role {
 						failedHostIPs[l3.RemoteIPAddress] = struct{}{}
-						failedHostNames = append(failedHostNames, hostname)
+						failedHostLatencies = append(failedHostLatencies, fmt.Sprintf(" %s (%.3fms)", hostname, l3.AverageRTTMs))
 					}
 				}
 			}
 		}
 	}
-	if len(failedHostNames) > 0 {
-		return ValidationFailure, failedHostNames, nil
+	if len(failedHostLatencies) > 0 {
+		return ValidationFailure, failedHostLatencies, nil
 	}
 	return ValidationSuccess, nil, nil
 }
 
+const (
+	lessThanOr = "less than or"
+	equals     = "equals"
+)
+
+func comparisonBuilder(value float64) string {
+	if value > 0 {
+		return fmt.Sprintf("%s %s", lessThanOr, equals)
+	}
+	return equals
+}
+
 func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validationContext, status ValidationStatus) string {
 	switch status {
 	case ValidationSuccess:
 		return "Network latency requirement has been satisfied."
 	case ValidationFailure:
-		_, hostNames, err := v.validateNetworkLatencyForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
+		_, hostLatencies, err := v.validateNetworkLatencyForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
 		if err != nil {
 			return fmt.Sprintf("Error while attempting to validate network latency: %s", err)
 		}
-		sort.Strings(hostNames)
-		return fmt.Sprintf("Network latency requirements of less or equals than %.3f ms not met for connectivity between %s and %s.", *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs, c.host.ID, strings.Join(hostNames, ","))
+		return fmt.Sprintf("Network latency requirements of %s %.3fms not met for connectivity between %s and%s.", comparisonBuilder(*c.clusterHostRequirements.Total.NetworkLatencyThresholdMs), *c.clusterHostRequirements.Total.NetworkLatencyThresholdMs, c.host.ID, strings.Join(hostLatencies, ","))
```

Comment:
Maybe like this?
```
Network latency requirements of less than or equals 100.000ms not met for connectivity between host "929834ec-78fa-4f39-b38b-d1e32adb8cb7" and hosts: "master-1" (200.000ms), "master-2" (200.000ms).
```
just so it will be more understandable

Reply:
I left a space between the figure and the unit (ms), and trimmed one digit from the decimal parts to avoid confusion. Maybe trim the space?
`100.00 ms` or `100.00ms`?
Which one to choose from @lalon4?